src/main/sensors/compass.c

```diff
@@ -75,11 +77,11 @@
 #define P0 1.0e2f        // value to initialize P(0) = diag([P0, P0, P0, P0]), typically in range: (1, 1000)
 
 #define CALIBRATION_WAIT_US (15 * 1000 * 1000)               // wait for movement to start and trigger the calibration routine in us
-#define GYRO_NORM_SQUARED_MIN sq(DEGREES_TO_RADIANS(450.0f)) // minimal value that has to be reached so that the calibration routine starts in (rad/sec)^2,
+#define GYRO_NORM_SQUARED_MIN sq(DEGREES_TO_RADIANS(350.0f)) // minimal value that has to be reached so that the calibration routine starts in (rad/sec)^2,
```

Comment:
(This is only because some users don't understand how to start calibration and insist on taping gyro)

Reply:
Yes, that's true.  I'm updating the documentation.  But 350 is quite readily achievable, with about the right amount of effort.  
If the user get more effective feedback about the starting of the calibration, they will learn what to do much better.
I had one user almost go nuts bashing their frame against a hard object :-(